common/scheme.go

```diff
@@ -20,5 +20,8 @@ type IScheme interface {
 	GetFormat() AttestationFormat
 	GetTrustAnchorID(token *AttestationToken) (string, error)
 	ExtractEvidence(token *AttestationToken, trustAnchor string) (*ExtractedEvidence, error)
-	GetAttestation(ec *EvidenceContext, endorsements string) (*Attestation, error)
+	GetAttestation(ec *EvidenceContext, endorsements []string) (*Attestation, error)
```

Comment:
Noticed @thomas-fossati  has changed the Interface for GetAttestation as well. Does that impact Verifier code @setrofim ?

Reply:
Not because the verifier doesn't interact with the scheme directly, but rather calls the `ITrustedServicesClient.Getattestation(*AttestationToken)` , the servier-side implementation for which then invokes the `IScheme.GetAttestation(*EvidenceContext, []string)`.